src/tests/stats/test_mean.f90

```diff
@@ -67,7 +78,7 @@ program test_mean
 
 !dp rank 4
 allocate(d4(size(d,1),size(d,2),3,9))
-d4 = 1.
+d4 = -1._dp
```

Comment:
Your change is correct, but even better is just:
```suggestion
d4 = -1
```
Then the precision is figured out automatically.

Reply:
modified